src/nextjournal/clerk/sci_viewer.cljs

```diff
@@ -185,21 +185,22 @@
   (map-indexed (fn [idx x]
                  (inspect (update opts :path conj idx) x))))
 
-(defn coll-viewer [{:keys [open close]} xs {:as opts :keys [path viewer !expanded-at]}]
-  (html (let [expanded? (@!expanded-at path)]
+(defn coll-viewer [xs {:as opts :keys [path viewer !expanded-at]}]
+  (html (let [expanded? (@!expanded-at path)
+              {:keys [opening-paren closing-paren]} viewer]
           [:span.inspected-value.whitespace-nowrap
            {:class (when expanded? "inline-flex")}
            [:span
             [:span.bg-opacity-70.whitespace-nowrap
              (when (< 1 (count xs))
                {:on-click (partial toggle-expanded !expanded-at path)
                 :class "cursor-pointer bg-indigo-50 hover:bg-indigo-100 hover:rounded-sm border-b border-gray-400 hover:border-gray-500"})
-             open]
+             opening-paren]
             (into [:<>]
                   (comp (inspect-children opts)
-                        (interpose (if expanded? [:<> [:br] nbsp (when (= 2 (count open)) nbsp)] " ")))
+                        (interpose (if expanded? [:<> [:br] nbsp (when (= 2 (count opening-paren)) nbsp)] " ")))
                   xs)
-            (into [:<>] (:closing-parens viewer close))]])))
+            (cond->> closing-paren (list? closing-paren) (into [:<>]))]])))
```

Comment:
I kind of need the `list?` check in order to accomodate the case e.g.

```clj
{:pred vector? :render-fn 'v/coll-viewer :closing-paren [:span.red "]"] ...
```
in the non-described case.